spring-integration-core/src/main/java/org/springframework/integration/aggregator/FluxAggregatorMessageHandler.java

```diff
@@ -277,6 +289,7 @@ private Mono<Message<?>> messageForWindowFlux(Flux<Message<?>> messageFlux) {
 								.build());
 	}
 
+	@NullUnmarked
 	private static Integer sequenceSizeHeader(Message<?> message) {
```

Comment:
The result of this method can be null, so:
```
private static @Nullable Integer sequenceSizeHeader(Message<?> message)
```
instead of `@NullUnmarked`

Reply:
With the change applied, It still exhibits the same behavior as enumerated here: https://github.com/spring-projects/spring-integration/pull/10097#discussion_r2141065674